package.json

```diff
@@ -16,6 +16,8 @@
     "start": "node dist/index.js"
   },
   "dependencies": {
+    "cross-fetch": "^3.1.4",
```

Comment:
You could also use node-fetch, it's the more popular alternative: https://www.npmtrends.com/cross-fetch-vs-node-fetch
But cross-fetch is also just fine.